src/libraries/src.proj

```diff
@@ -46,4 +46,12 @@
              Properties="$(TraversalGlobalProperties)" />
   </Target>
 
+  <!-- Wasm runtimes depend on the mono runtime and libs to be built, so they can only be built here -->
+  <Target Name="BuildWasmPackageDependency"
+          Condition="'$(TargetOS)' == 'Browser'"
+          AfterTargets="Build">
+    <MSBuild Projects="$(MonoProjectRoot)\wasm\wasm.proj"
```

Comment:
Why are we building a mono project from within libraries?

Reply:
WASM is a bit special, it needs to run a processor over the whole BCL to find p/invokes and generate a table with names and signatures from them. This seemed like a good place to do that.

Reply:
Wouldn't it make more sense to do that in a mono subset instead which runs after libraries?

Reply:
Sure, if you show me how to do that :)

Reply:
Sure, if you show me how to do that :)

Reply:
I'm fine if you do that in a follow-up if you prefer. I will submit a PR make it easier to run after some subsets.